packages/python/chart-studio/chart_studio/presentation_objs/presentation_objs.py

```diff
@@ -973,6 +973,7 @@ def _markdown_to_presentation(self, markdown_string, style, imgStretch):
             if slide.count("```") % 2 != 0:
                 raise _plotly_utils.exceptions.PlotlyError(CODE_ENV_ERROR)
 
+            wdw_size = len("```")
```

Comment:
Why not simply set `wdw_size = 3`? or why even create this variable, it's being used only in one place?

Reply:
I tried to change as little as possible. There might be a reason (readability, further use in the future) why this variable was there before.